config/gta3/cleo.xml

```diff
@@ -0,0 +1,958 @@
+      <Args>
+        <Arg Type="INT" Entity="PLAYER"/>
+      </Args>
+    </Command>
+    <Command ID="0x4a8" Name="IS_PLAYER_IN_ANY_BOAT" Hash="0xe96e0fea">
+      <Args>
+        <Arg Type="INT" Entity="PLAYER"/>
+      </Args>
+    </Command>
+    <Command ID="0x4aa" Name="IS_PLAYER_IN_ANY_HELI" Hash="0x8cc3fb48">
+      <Args>
+        <Arg Type="INT" Entity="PLAYER"/>
+      </Args>
+    </Command>
+    <Command ID="0x47e" Name="IS_PLAYER_ON_ANY_BIKE" Hash="0x36f8a3e4">
+      <Args>
+        <Arg Type="INT" Entity="PLAYER"/>
+      </Args>
+    </Command>
+    <Command ID="0x485" Name="IS_PC_VERSION" Hash="0x5f927dd1"/>
+    <Command ID="0x59a" Name="IS_AUSTRALIAN_GAME" Hash="0x40172dc1"/>
+    <Command ID="0x601" Name="IS_KEY_PRESSED_WITH_SENSITIVITY" Hash="0x662d65d3">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+      </Args>
+    </Command>
+    <Command ID="0x602" Name="EMULATE_KEY_PRESS_WITH_SENSITIVITY" Hash="0x25eda09c">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+      </Args>
+    </Command>
+    <Command ID="0x603" Name="IS_CAMERA_IN_WIDESCREEN_MODE" Hash="0xcd6ad16b"/>
+    <Command ID="0x604" Name="GET_MODEL_ID_OF_A_WEAPON" Hash="0x60774d21">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x605" Name="GET_WEAPON_ID_OF_A_MODEL" Hash="0x887e993a">
+      <Args>
+        <Arg Type="INT" Enum="DEFAULTMODEL"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x606" Name="SET_MEMORY_OFFSET" Hash="0x7cab85ce">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+      </Args>
+    </Command>
+    <Command ID="0x607" Name="GET_CURRENT_WEATHER" Hash="0xcf0b6bc6">
+      <Args>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x608" Name="DISPLAY_TEXT_WITH_STRING" Hash="0x51e4cae3">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+      </Args>
+    </Command>
+    <Command ID="0x609" Name="DISPLAY_TEXT_WITH_STRING_FORMATTED" Hash="0xdf24eba1">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+        <Arg Type="PARAM" Optional="true" AllowTextLabel="true" PreserveCase="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xa8c" Name="WRITE_MEMORY" Hash="0xc3cc351e">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="PARAM"/>
+        <Arg Type="INT"/>
+      </Args>
+    </Command>
+    <Command ID="0xa8d" Name="READ_MEMORY" Hash="0x7fc4ddcc">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="PARAM" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xa8e" Name="INT_ADD" Hash="0x4c2b352a">
```

Comment:
Not sure if worth adding, this is legacy from SCM Assemblers needing a easy way to type a `x = a + b` expression. Nice name though.